dblookupext/historyRepository.go

```diff
@@ -265,7 +291,29 @@ func (hr *historyRepository) GetMiniblockMetadataByTxHash(hash []byte) (*Miniblo
 		return nil, err
 	}
 
-	return hr.getMiniblockMetadataByMiniblockHash(miniblockHash)
+	hashSize := hr.hasher.Size()
+	if len(miniblockHash) == hashSize {
+		return hr.getMiniblockMetadataByMiniblockHash(miniblockHash)
+	}
+
+	prefixedHashSize := hashSize + 1
+	if len(miniblockHash) != prefixedHashSize {
+		return nil, ErrInvalidHashSize
+	}
+
+	prefix := miniblockHash[0]
+	actualHash := miniblockHash[1:]
+	switch prefix {
+	case byte(RelayedTxHash):
```

Comment:
I don't think this is correct, asyou would call hr.getMiniblockMetadataByMiniblockHash(miniblockHash) but miniBlockHash is the relay tx hash.
 
for the [relayTxId]innertxhash you have saved the relayTxHash so the miniBlockHash here is in fact a relayTxHash. This means here you would have to call again the GetMiniBlockMetaDataByTxHash using this time the relaytxhash as argument, right?